config/samples/databricks_v1alpha1_djob.yaml

```diff
@@ -1,22 +1,23 @@
 apiVersion: databricks.microsoft.com/v1alpha1
 kind: Djob
 metadata:
-  name: djob-sample
+  name: djob-sample-1
 spec:
   # This spec is directly linked to the JobSettings structure
   # https://docs.databricks.com/api/latest/jobs.html#jobsettings
   new_cluster:
     spark_version: 5.3.x-scala2.11
     node_type_id: Standard_D3_v2
-    num_workers: 3
-  libraries:
-    - jar: 'dbfs:/my-jar.jar'
-    - maven:
-        coordinates: 'org.jsoup:jsoup:1.7.2'
+    num_workers: 10
+  # libraries:
+    # # - jar: 'dbfs:/my-jar.jar'
+    # - maven:
+        # coordinates: 'org.jsoup:jsoup:1.7.2'
   timeout_seconds: 3600
-  max_retries: 1
+  max_retries: 4
   schedule:
     quartz_cron_expression: 0 15 22 ? * *
     timezone_id: America/Los_Angeles
-  spark_jar_task:
-    main_class_name: com.databricks.ComputeModels
+  notebook_task:
+    notebook_path: /Users/amr.asem@dataexchange.work/djob_run_test_amr_clone
```

Comment:
Could you please update the notebook path to notebook_path: "/samples/basic1" ?
We have some samples here => github.com/azure-databricks-operator/docs/samples

Reply:
is this path accessible "github.com/azure-databricks-operator/docs/samples" , how can I access it?

Reply:
got it, thanks